maths/trapezoidal_rule.py

```diff
@@ -1,47 +1,103 @@
 """
 Numerical integration or quadrature for a smooth function f with known values at x_i
 
-This method is the classical approach of suming 'Equally Spaced Abscissas'
+This method is the classical approach of summing 'Equally Spaced Abscissas'
```

Comment:
```suggestion
The trapezoidal rule is the classical approach of summing 'Equally Spaced Abscissas'
```
Let's rephrase the header docstring to name the rule directly. I'm not sure why the original author called it "method 1".